### DISCUSSION TITLE
Repository langauge not detected

### DISCUSSION THREAD

**Discussion by Marc-SA** [github-linguist/linguist#5801] [Classification] [answered]
Hello, GitHub is not displaying any language for my repository [repo](https://github.com/Marc-SA/AOSEP) please help, thanks

**Comment by lildude** [comment, marked as answer]
🤔 it looks like the background job didn't run for some reason. I've seen this randomly happen when all code added to a repo is done in a single commit like you've done. Push another change and things should be good again.
